_includes/README.md

```diff
@@ -19,9 +19,10 @@ be better?
 These pages also host the [Git Rev News](https://git.github.io/rev_news/),
 see the [About Git Rev News](https://git.github.io/rev_news/rev_news/) page.
 
-If you want push access, contact peff@peff.net and provide your GitHub
-username. You may also send patches by mail (and feel free to cc
-git@vger.kernel.org if appropriate).
+If you want push access, contact Christian Couder
+\<<chriscool@tuxfamily.org>\> or Taylor Blau \<<me@ttaylorr.com>\> and
```

Comment:
My christian.couder@gmail.com address is probably better than chriscool@tuxfamily.org.

Reply:
Works for me. I just ran my usual `:Git who christian` which picked out the last email you committed with in this project. But I'll switch to [christian.couder@gmail.com](mailto:christian.couder@gmail.com) instead!